### PR TITLE
Table.ExpandListColumn -- documenting behavior on nested table columns

### PR DESCRIPTION
**Pull request by bgribaudo**

### CODE REVIEW

**Comment by Jak-MS** [comment]
@DougKlopfenstein 
- Can you review this PR?
- IMPORTANT: When this content is ready to merge, you must add `#sign-off` in a comment or the approval may get overlooked.


#label:"aq-pr-triaged"
@MicrosoftDocs/public-repo-pr-review-team

File: query-languages/m/table-expandlistcolumn.md [hunk 1]
```diff
@@ -12,7 +12,7 @@ Table.ExpandListColumn(<b>table</b> as table, <b>column</b> as text) as table
   
 ## About
 
-Given a `table`, where a `column` is a list of values, splits the list into a row for each value. Values in the other columns are duplicated in each new row created.
+Given a `table`, when `column` contains a list of values, splits the list into a row for each value. If, instead, `column` contains a nested table, treats the nested table as though it were a list of records, which it then splits into a row for each value. Values in the other columns are duplicated in each new row created.
```
Comment (ehrenMSFT) [inline query-languages/m/table-expandlistcolumn.md:15]:
```suggestion
Given a `table` where `column` contains a list of values, splits the list into a row for each value. Values in the other columns are duplicated in each new row created. This function can also expand nested tables by treating them as lists of records.
```

File: query-languages/m/table-expandlistcolumn.md [hunk 2]
```diff
@@ -27,12 +27,28 @@ Table.ExpandListColumn(
 )
 ```
 
+## Example 2
+
+Split the nested table column [Components] in the table.
```
Comment (ehrenMSFT) [inline query-languages/m/table-expandlistcolumn.md:32]: Nix "in the table", both here and above in the existing example.

```suggestion
Split the nested table column [Components].
```

**Review by ehrenMSFT — CHANGES_REQUESTED** [review]

File: query-languages/m/table-expandlistcolumn.md (hunk 2 above)
Comment (bgribaudo) [reply query-languages/m/table-expandlistcolumn.md]: Good idea. Done.

**Review by ehrenMSFT — APPROVED** [review]

**Comment by DougKlopfenstein** [comment]
#sign-off

**Pull request merged**